lib/pka_dev.c

```diff
@@ -50,6 +50,30 @@
 #include "pka_dev.h"
 
 #ifdef __KERNEL__
+
+// Personalization string "NVIDIA-MELLANOX-BLUEFIELD-TRUE_RANDOM_NUMBER_GEN"
+uint32_t pka_trng_drbg_ps_str[] = {0x4e564944, 0x49412d4d, 0x454c4c41, 0x4e4f582d,

```

Comment:
To be consistent with the coding style of the PKA library, could you do the following?

uint32_t pka_trng_drbg_ps_str[] = {
  value1, value2, value3, value4,
  value5, etc.
};

Reply:
Yes, will do.

Reply:
Is this the latest commit? Or should I expect a new one?

Reply:
There will be one more rebased on test circuits. Thanks

Reply:
I have added the latest commit, rebased on top of TRNG test circuits commit.